setup.py

```diff
@@ -54,6 +54,6 @@
     extras_require={
         'dev': ['flake8', 'mypy'],
         'unit-test': ['pytest', 'pytest-cov', 'pytest-mock'],
-        'system-test': ['systest >= 5.5.0']
+        'system-test': ['pytest', 'pytest-cov', 'pytest-mock']
```

Comment:
We probably shouldn't include the ability to "mock" in the system tests. If you are mocking you've probably gone wrong somewhere. System tests should be testing the whole system, not stubbing bits out.

Reply:
Good point - I'll remove it.  While we're on the topic what about the coverage part?  Initially I was thinking it's useful to know how much of the codebase the system tests cover, but unless it's extremely clever I've a feeling it might not tell us much (seeing as most of the actual testing is happening inside `suprocess` calls to the various applications